charms/worker/k8s/src/charm.py

```diff
@@ -399,7 +400,7 @@ def _revoke_cluster_tokens(self, event: ops.EventBase):
         """
         log.info("Garbage collect cluster tokens")
         to_remove = None
-        if self._stored.is_dying:
+        if self._stored.is_dying is True:
```

Comment:
again -- this is a typing thing more than a python issue.  pylint doesn't seem to like implicit type conversions to booleans?